docs/blog.md

```diff
@@ -72,6 +72,12 @@ pageClass: routes
 
 <Route author="xyqfer" example="/leemeng" path="/leemeng"/>
 
+## Love the Problem
+
+Ash Maurya's blog
+
+<Route author="james-tindal" example="/blogs/love-the-problem" path="/blogs/love-the-problem"/>
```

Comment:
```suggestion
<RouteEn author="james-tindal" example="/blogs/love-the-problem" path="/blogs/love-the-problem"/>
```
Apart from the changes, you should add this documentation in docs/en/blog.md instead as it’s in English.

Reply:
This is blocked by #8346